Dockerfile

```
@@ -60,6 +60,11 @@ RUN a2enmod authz_core deflate expires headers mime rewrite setenvif
 # Set DocumentRoot
 RUN sed 's:/var/www/html:/var/www/application:' -i /etc/apache2/sites-available/000-default.conf
 
+# Allow to listen on a different port than 80
+ENV APACHE_PORT=80
+RUN sed 's:<VirtualHost \*\:80>:<VirtualHost \*\:${APACHE_PORT}>:' -i /etc/apache2/sites-available/000-default.conf
```

hier würde ich das trennzeichen von `sed`-`substitute` auf etwas ändern, was nicht in den strings vorkommt..

```suggestion
RUN sed 's/<VirtualHost \*:80>/<VirtualHost \*:${APACHE_PORT}>/' -i /etc/apache2/sites-available/000-default.conf
```